parm/ccpp_regional_c768_FA.nml.IN

```diff
@@ -129,6 +129,9 @@
        pre_rad        = .false.
        ncld           = 5
        imp_physics    = 15
```

Comment:
I've been doing the same thing as @mzhangw, editing this namelist for my HWRF physics schemes. Should we consider changing the name since it will be valid for many (all?) HWRF schemes (and not just the FA scheme)?

Reply:
we can consolidate all HWRF namelist options once all schemes are ready.

Reply:
@grantfirl if you like, please do that as part of the Noah LSM PR (when you pull in the updates from these PRs before the final testing). Thanks!